clients/drcachesim/tracer/physaddr.cpp

```diff
@@ -93,8 +98,12 @@ physaddr_t::init()
 #ifdef LINUX
     // Some threads may not do much, so start out small.
     constexpr int V2P_INITIAL_BITS = 9;
-    hashtable_init_ex(&v2p_, V2P_INITIAL_BITS, HASH_INTPTR, /*strdup=*/false,
-                      /*synch=*/false, nullptr, nullptr, nullptr);
+    // The hashtable lookup performance is important.
+    // A non-open-address hashtable is about 3x slower due to the extra
+    // loads, and higher resize thresholds are also slower.
```

Comment:
Aren't the number of loads the same? It's just that the loads have better locality, right?

Reply:
No, the regular table stores a pointer to the payload it has to de-reference, so there is an extra load.  The open address here has the data inlined.

Reply:
Added a mention of the inlined payload to the comment.